OrbitGl/GraphTrack.h

```diff
@@ -19,9 +20,12 @@ class GraphTrack : public Track {
   void Draw(GlCanvas* canvas, PickingMode /*picking_mode*/) override;
   [[nodiscard]] float GetHeight() const override;
   void AddValue(double value, uint64_t time);
-  [[nodiscard]] double GetValueAtTime(uint64_t time, double default_value = 0) const;
+  [[nodiscard]] std::optional<std::pair<uint64_t, double> > GetLastMeasure(uint64_t time) const;
```

Comment:
nit. I'm not sure about "Measure", I don't think it represents what is being returned. GetLastPoint maybe? Or GetPointForTime?

Reply:
I don't like point, because it isn't a point. It's a pair of (value, time), where after we can make some mathematical operations to get the x-y coordinates of a future point. So, for me it's the last measure. 

What do you think about GetPreviousValueAndTime?